.github/workflows/update-lit.yml

```diff
@@ -0,0 +1,50 @@
+name: Package Migration
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Do we only want this run on PR? might be good to run this on commit to branches like `master` or feature branches to ensure latest package compatibility.

Reply:
I initially wrote this logic in a way that every time a new package update is detected, dependabot will send a pr and this script runs on that to migrate all of the existing packages to the latest version and push its commit to it.

But if this needs to run on all of the push commits to master and feature or if we want to run this on every pr made to master to auto-update a new examples package if by chance an older one was used and then run tests. I can change it that way then. Lemme know what seems a better approach

Reply:
I think running on every pr to master, and on commit to master, such that if someone who has access to do direct pushes to master does so. We can still run logic to check on version compatibility.

Reply:
that makes sense, for now I'm leaving it to master but you can suggest a commit if you feel like

Reply:
This would be based on this repository git ops strategy. So feel free to apply what I suggested to what works best for your workflows.